src/main/java/me/totalfreedom/totalfreedommod/command/Command_seed.java

```diff
@@ -0,0 +1,33 @@
+package me.totalfreedom.totalfreedommod.command;
+
+import org.bukkit.Bukkit;
+import org.bukkit.World;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
```

Comment:
missing permissions annotation, commands without these will cause the whole plugin to shit the bed